userspace/libsinsp/sinsp.cpp

```diff
@@ -1458,14 +1480,14 @@ threadinfo_map_t::ptr_t sinsp::get_thread_ref(int64_t tid, bool query_os_if_not_
 
 		m_n_proc_lookups++;
 
-		if(m_n_proc_lookups == m_max_n_proc_lookups)
+		if(m_n_proc_lookups == get_max_n_proc_loopkups())
```

Comment:
As this method does some calculations (and maybe even syscalls to `gettimeofday()` if we're unlucky), maybe we should call it just once in this method and use the cached value?

Reply:
agree